packages/engine/Source/DataSources/BillboardGraphics.js

```diff
@@ -303,7 +302,7 @@ Object.defineProperties(BillboardGraphics.prototype, {
    * @type {Property|undefined}
    */
   pixelOffsetScaleByDistance: createPropertyDescriptor(
-    "pixelOffsetScaleByDistance"
+    "pixelOffsetScaleByDistance",
```

Comment:
It seems like you have the wrong version of `prettier` installed or your editor configuration is wrong. These changes are distracting from the intended change of this PR (I'm working on the update in #12206). Can you do a `npm install` and make sure to run `npm run prettier`? Also probably worth checking that the pre-commit hook is working as intended so these can't even be committed.

Reply:
Okay. I see the pre-commit hook. There was probably something I did wrong. But it looks like it is preventing commits now. 

I ran prettier and merged the changes into my pull request.

<img width="638" alt="image" src="https://github.com/user-attachments/assets/1d61b49a-89f1-4088-8e0e-e77dd2227868">